### ISSUE TITLE
Documentation of Suspect false pattern generation

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/CORRECTIONS#25]
Let me document the method I have adopted to find the suspected wrong entries in #2. 
Code for checking is attached <a href="https://github.com/drdhaval2785/SanskritSpellCheck">here</a>. 
<a href="https://docs.google.com/document/d/1G4HoDz9nuj2GPeHQopNVSnDEGrnXtoAuXFugj4sQHZg/edit?usp=sharing">Google doc</a> for logic behind approach and instructions for dictionary /testers.
Video tutorial for running PHP script- http://youtu.be/qLqYUZUGM6M
Video tutorial for correction submission (based on wrong entries found from suspectfalse.html) on this corrections page - https://www.youtube.com/watch?v=rKZ_OsSHwsY

In short - the logic is as follows
1. We check the base dictionary for different Vowel and consonant patterns like VV, VCV, VCCV, VCCCV etc. (We presume that the base dictionary has correct entries - which need not be true).
2. We check the dictionary to be tested for different vowel and consonant patterns like 1 above.
3. If the pattern is not found in 1, but found in 2 - we note that in suspectfalse.html.
4. We check those entries from dictionary scans and verify.
5. If wrong word is detected, it is intimated to the webmaster here.

**Comment by drdhaval2785** [comment]
The method has worked and is near its end of exploitation.
So, closing this issue to remove clutter from the plate.

**Comment by funderburkjim** [comment]
And a venerable method it has been!

**Comment by gasyoun** [comment]
Too early to RIP it I guess.
